doc/api/http.md

```diff
@@ -747,6 +747,10 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: in most operating systems, listening to any IPv6 address (`'::'`) may
```

Comment:
Good point, @gibfahn , but we should define the term "unspecified", perhaps link to https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address as well markup the word with `_..._`, so its clear that its a technical term. @QianJin2013 `INADDR_ANY` and `::` are not the same thing, but I agree, _any_ probably cames from the IPv4 macro name, which doesn't make sense when talking about an IPv6 address!